packages/react/src/forms/ControlGroup/ControlGroup.module.css

```diff
@@ -0,0 +1,55 @@
+.ControlGroup__container {
+  display: flex;
+  flex-direction: column;
+  gap: var(--base-size-8);
+  font-family: var(--brand-body-fontFamily);
+  font-size: var(--brand-text-size-100);
+  border: none;
+  margin: 0;
+  padding: 0;
+}
+
+.ControlGroup__label {
+  margin: 0;
+  padding: 0;
+  margin-block-end: var(--base-size-8);
+}
+
+.ControlGroup__validation {
+  display: flex;
+  gap: var(--base-size-4);
+  font-weight: var(--base-text-weight-medium);
+}
+
+.ControlGroup__validation--success {
+  color: var(--brand-color-success-fg);
+}
+
+.ControlGroup__validation--error {
+  color: var(--brand-color-error-fg);
+}
+
+.ControlGroup__validation-icon--success {
+  position: relative;
+  top: -1px;
+}
+
+.ControlGroup__validation-icon--error {
+  position: relative;
+  top: -0.5px;
+}
+
+.ControlGroup__validation--animate-in {
+  animation: 170ms ControlGroupValidationFadeIn cubic-bezier(0.44, 0.74, 0.36, 1);
```

Comment:
Non-blocking: 

Appreciate these are probably from FormControl. We have newer animation-specific variables now, but none that match directly to this.

@joshfarrant, how close are these values to what's above? Can we sub them out? Can also do this in a separate PR.

![Uploading Screenshot 2024-12-03 at 11.55.34.png…]()

Reply:
Yes this was pulled from FormControl. You can see the three curves below. 

Our curve:
![image](https://github.com/user-attachments/assets/bc54ec81-6b9e-47a0-8f33-c74a3b173482)

vs default:
![image](https://github.com/user-attachments/assets/cfd46f23-72ff-44e8-924d-d2238dbca13f)

vs glide:
![image](https://github.com/user-attachments/assets/9a9cc699-f839-45bb-84b2-bd2b9242261b)

I'd say that glide is probably the closest. Happy to switch this out

Reply:
I've just gone ahead and updated it to `brand-animation-duration-glide`